docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,489 @@
+- :math:`\alpha_0` and :math:`\alpha_1` are the qubit anharmonicities, 
+- :math:`J` is the coupling strength, 
+- :math:`r_0` and :math:`r_1` are the Rabi strengths, and :math:`s_0(t)` and :math:`s_1(t)` are the
+  drive signals, 
+- :math:`a_j` and :math:`a_j^\dagger` are the lowering and raising operators for qubit :math:`j`,
+  and
+- :math:`N_0` and :math:`N_1` are the number operators.
+
+.. jupyter-execute::
+
+    import numpy as np
+    
+    dim = 3
+    
+    v0 = 4.86e9
+    anharm0 = -0.32e9
+    r0 = 0.22e9
+    
+    v1 = 4.97e9
+    anharm1 = -0.32e9
+    r1 = 0.26e9
+    
+    J = 0.002e9
+    
+    a = np.diag(np.sqrt(np.arange(1, dim)), 1)
+    adag = np.diag(np.sqrt(np.arange(1, dim)), -1)
+    N = np.diag(np.arange(dim))
+    
+    ident = np.eye(dim, dtype=complex)
+    full_ident = np.eye(dim**2, dtype=complex)
+    
+    N0 = np.kron(ident, N)
+    N1 = np.kron(N, ident)
+    
+    a0 = np.kron(ident, a)
+    a1 = np.kron(a, ident)
+    
+    a0dag = np.kron(ident, adag)
+    a1dag = np.kron(adag, ident)
+    
+    
+    static_ham0 = 2 * np.pi * v0 * N0 + np.pi * anharm0 * N0 * (N0 - full_ident)
+    static_ham1 = 2 * np.pi * v1 * N1 + np.pi * anharm1 * N1 * (N1 - full_ident)
+    
+    static_ham_full = static_ham0 + static_ham1 + 2 * np.pi * J * ((a0 + a0dag) @ (a1 + a1dag))
+    
+    drive_op0 = 2 * np.pi * r0 * (a0 + a0dag)
+    drive_op1 = 2 * np.pi * r1 * (a1 + a1dag)
+
+Construct the :class:`.Solver` using the model details, including parameters necessary for pulse
+simulation. See the :class:`.Solver` documentation, as well as the :ref:`tutorial example <pulse
+solver>` for more details.
+
+.. jupyter-execute::
+
+    from qiskit_dynamics import Solver
+    
+    # build solver
+    dt = 1/4.5e9
+    
+    solver = Solver(
+        static_hamiltonian=static_ham_full,
+        hamiltonian_operators=[drive_op0, drive_op1, drive_op0, drive_op1],
+        rotating_frame=static_ham_full,
+        hamiltonian_channels=["d0", "d1", "u0", "u1"],
+        channel_carrier_freqs={"d0": v0, "d1": v1, "u0": v1, "u1": v0},
+        dt=dt,
+    )
+
+Next, instantiate the :class:`.DynamicsBackend`. The ``solver`` is used for simulation,
+``subsystem_dims`` indicates how the full system decomposes for measurement data computation, and
+``solver_options`` are consistent options used by :meth:`.Solver.solve` when simulating the
+differential equation. The full list of allowable ``solver_options`` are the arguments to
+:func:`.solve_ode`.
+
+Note that, to enable the internal automatic jit-compilation, we choose a JAX integration method.
+
+.. jupyter-execute::
+
+    from qiskit_dynamics import DynamicsBackend
+    
+    # Consistent solver option to use throughout notebook
+    solver_options = {"method": "jax_odeint", "atol": 1e-6, "rtol": 1e-8}
+    
+    backend = DynamicsBackend(
+        solver=solver,
+        subsystem_dims=[dim, dim], # for computing measurement data
```

Comment:
What happens if you set [2, 2] here? f-state is measured as 1?

Reply:
No, an error will be raised in that case, as there is a validation check ensuring that `np.prod(subsystem_dims)` is equal to `solver.model.dim`. Customization of how measurement outcomes are computed can be done with other `DynamicsBackend` options, but `subsystem_dims` is purely meant to be an exact specification of how the model in `solver` breaks into subsystems.